deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
 spec:
   requests:
   - registry: common-service
-    registryNamespace: ibm-common-services
```

Comment:
I think we can leave this in the CR, this can help others understand this CR.

Reply:
Okay, add it back